apps/wallet/src/ui/app/components/accounts/AccountItem.tsx

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { formatAddress } from '@mysten/sui.js/utils';
+
+import cn from 'classnames';
+import { type ReactNode } from 'react';
+import { AddressLink } from '../explorer-link';
+import { Text } from '_src/ui/app/shared/text';
+
+interface AccountItemProps {
+	name?: string | null;
+	address: string;
+	icon?: ReactNode;
+	after?: ReactNode;
+	disabled?: boolean;
+	selected?: boolean;
+}
+
+export function AccountItem({
+	icon,
+	after,
+	disabled,
+	selected,
+	name,
+	address,
+	...props
```

Comment:
do we need this? `...props`

Reply:
I think it's a bit confusing the way I've separate out the components, but this is required so that we can use this as a child of `ToggleGroup.Item` with the `asChild` prop.  https://www.radix-ui.com/docs/primitives/guides/composition#your-component-must-spread-props